### ISSUE TITLE
 NTR: Defective GlcNAc-PI synthesis 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4661]
**Preferred term label:**
Defective GlcNAc-PI synthesis 

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**
A defect in the synthesis of GlcNAc-PI; Glycosylphosphatidylinositol (GPI) is biosynthesized by the sequential addition of carbohydrates to phosphatidylinositol (PI). In the first two reactions, GlcNAc is transferred from UDP-GlcNAc to PI and then deacetylated to form GlcNAc-PI and GlcN-PI, respectively. 
Source: PMID:8387504

**Parent term (use hpo.jax.org/app)**
HP_0011017 'Abnormal cellular physiology'
_Not sure if this is the best parent?_

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/300818?search=300818&highlight=300818

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by pnrobinson** [comment]
@nicolevasilevsky  This does not sound like a phenotypic observation -- it is an inference. What is being observed here? Too low x? Abnormal Y test? etc.

**Comment by nicolevasilevsky** [comment]
The OMIM record says " Defective GlcNAc-PI synthesis" on the clinical synposis, and the description says "Complementation analysis by somatic cell hybridization with GPI-deficient mutant cell lines showed that these PNH cell lines belonged to complementation class A, which is known not to synthesize GlcNAc-PI."

**Comment by pnrobinson** [comment]
I see what you mean, but this was the result of a complicated scientific experiment and not a clinical measurement. No future patient will be analyzed in this way (hybrid cell lines etc). So I do not think this is a great HPO term. It should be described in some future pathophysiology ontology.

**Comment by nicolevasilevsky** [comment]
Ok! I'll close this ticket.
